openassessment/xblock/static/js/src/lms/oa_staff_area.js

```diff
@@ -471,6 +471,9 @@
                 view.baseView.unsavedWarningEnabled(false, view.FULL_GRADE_UNSAVED_WARNING_KEY);
                 view.staffGradeFormLoaded = false;
                 view.loadStaffGradeForm(!continueGrading);
+                if (continueGrading) {
```

Comment:
Note to reviewers: without scrolling up, it's not at all obvious what happened when you press "Submit assessment and continue grading". The problem was introduced in my last PR, where I changed the refresh model so we aren't re-rendering the entire staff area.